lib/decorators/api-extension.decorator.ts

```diff
@@ -9,10 +10,7 @@ export function ApiExtension(extensionKey: string, extensionProperties: any) {
   }
 
   const extensionObject = {
-    [extensionKey]:
-      typeof extensionProperties !== 'string'
-        ? { ...extensionProperties }
-        : extensionProperties
+    [extensionKey]: clone(extensionProperties)
```

Comment:
The tests should probably be changed (e.g. the e2e tests) to add an example with an array value

Reply:
Test case added